addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -81,6 +81,19 @@ def test_message_parse_body(self):
         res = self.env['mail.thread'].message_parse(self.from_string(test_mail_data.MAIL_MULTIPART_WEIRD_FILENAME))
         self.assertEqual(res['attachments'][0][0], '62_@;,][)=.(ÇÀÉ.txt')
 
+    def test_pdf_mime_type_aliasing(self):
```

Comment:
Try to order methods alphabetically in a given test class. Rationale is that they are executed alphabetically, so it makes sense to define them following this order, this helps finding tests / updating them in refactoring :)